test/specs/AutoRegisterDataProvideSpec.js

```diff
@@ -0,0 +1,21 @@
+describe("Auto register data-provide Tests", function() {
+  it("checks that the autoregister Slider was automatically registerd", function() {
```

Comment:
could you add `after()` block that calls the destroy() method on this slider just to ensure proper cleanup & no conflicts with other tests?

Other than that, this looks great! Thanks!